src/commands/view.py

```diff
@@ -115,3 +117,42 @@ def node_status(node):
             rows.sort(key=lambda x:x[0])
         display_table(headers, rows)
 
+    @view.group(name='tool-status', help='View the execution history of a single tool')
+    def tool_status():
+        pass
+
+    @click_tools.command(tool_status, exclude_interactive_only = True)
+    def tool_status_runner(config, _):
+        session = Session()
+        # Returns the most recent job for each node and the number of times the tool's been ran
+        # Refs: https://docs.sqlalchemy.org/en/latest/core/functions.html#sqlalchemy.sql.functions.count
+        #       https://www.w3schools.com/sql/func_sqlserver_count.asp
+        # => [(latest_job1, count1), (lastest_job2, count2), ...]
+        node_data = session.query(Job, func.count(Job.node))\
+                           .select_from(Batch)\
+                           .filter(Batch.config == config.path)\
+                           .join(Job.batch)\
+                           .order_by(Job.created_date.desc())\
+                           .group_by(Job.node)\
+                           .all()
+        if not node_data: raise ClickException('No jobs found for tool {}'.format(config.__name__()))
```

Comment:
The following is a code duplication of:
https://github.com/alces-software/adminware/blob/34970230f0e1268f86b9497a99f4fe5454ff583e/src/commands/view.py#L99-118

Only the first column is different, so there should be a way to cleanly handle it.